src/productcatalogservice/server.go

```diff
@@ -76,6 +80,30 @@ func init() {
 	if err != nil {
 		log.Warnf("could not parse product catalog")
 	}
+
+	//init runtime metrics
```

Comment:
is this using the prometheus client for Go? Should we not use the OTel library for Go instead?

Reply:
Agreed, should use otel library.